client/src/app/project-registration/autofill-project-form/autofill-project-form.component.ts

```diff
@@ -16,16 +18,19 @@ import {IngestService} from '../../shared/services/ingest.service';
 
 export class AutofillProjectFormComponent implements OnInit {
   publicationDoiCtrl: FormControl;
+  projectInCache$: Observable<Project>;
 
   constructor(private route: ActivatedRoute,
               private router: Router,
               private ingestService: IngestService,
-              private alertService: AlertService
+              private alertService: AlertService,
+              private cacheProjectService: ProjectCacheService
```

Comment:
since you followed my suggestion, you could also rename the variable to be projectCacheService :)